packages/query/src/block-processor.ts

```diff
@@ -52,6 +48,8 @@ interface QueryClientProps {
   querier: RootQuerier;
   indexedDb: IndexedDbInterface;
   viewServer: ViewServerInterface;
+  numeraires: Metadata[];
+  stakingTokenMetadata: Metadata;
```

Comment:
`numeraires` and `stakingTokenMetadata` are passed at initialization to avoid registry dependency for the block processor